src/components/views/settings/tabs/user/SecurityUserSettingsTab.js

```diff
@@ -341,8 +342,9 @@ export default class SecurityUserSettingsTab extends React.Component {
         }
 
         let privacySection;
-        if (Analytics.canEnable() || CountlyAnalytics.instance.canEnable()) {
-            privacySection = <React.Fragment>
+        if ((Analytics.canEnable() || CountlyAnalytics.instance.canEnable()) && SettingsStore.getValue(UIFeature.ShowShareAnalyticsInformation)) {
```

Comment:
ShowShareAnalyticsInformation is not required as we have already disabled analytics by prefixing pwik with DISABLED

Reply:
I have removed this one , I had done this to make it more secure but looking into other two conditions I found, if pwik config is not found then its not going to show what is shown in UI with this condition. So, technically we dont need it. Maybe I had over-engineered it. :). I think my approach was even if pwik is tweaked in config accidently, then the flag I introduced will always turn off that view and not display.